pyserini/2cr/beir.py

```diff
@@ -17,16 +17,22 @@
 import argparse
 import math
 import os
-import time
 import sys
+import time
 from collections import defaultdict
+from datetime import datetime
 from string import Template
-import pkg_resources
 
+import pkg_resources
 import yaml
 
 from ._base import run_eval_and_return_metric, ok_str, fail_str
 
+dense_threads = 16
```

Comment:
I think it might make more sense to move these constants into `pyserini/2cr/base` since they're reused a bunch. Your call though

Reply:
Otherwise, this looks good to me!

Reply:
I was thinking that I might want the ability to control on a per-regression basis, so I'll keep separate?

Please +1 the review so I can merge? Thanks!